src/app/workflow/snapshot-exporter-modal/snaphot-exporter-modal.component.html

```diff
@@ -61,15 +61,9 @@ <h4 mat-dialog-title>
       </div>
 
       <div mat-dialog-actions fxLayout="row" fxLayoutAlign="end">
-        <div
-          *ngIf="promptToConfirmSnapshot && state.overall === StepState.INITIAL"
-          fxLayout="row"
-          fxLayoutAlign="end"
-          fxFlex="50"
-          fxFlexOffset="50"
-        >
-          <span class="alert-warning pb-2 mb-2"
-            >Are you sure you want to snapshot this version, making it uneditable? This cannot be undone.</span
+        <div *ngIf="promptToConfirmSnapshot && state.overall === StepState.INITIAL" fxLayout="row" fxLayoutAlign="end" class="mb-3 w-100">
```

Comment:
This updates the styling of the confirmation message to match the one in the topic editing dialog. The text remains the same